--- a/src/Makefile
+++ b/src/Makefile
@@ -3,19 +3,14 @@ NAME 	= init
 OBJS 	= plpkexec.o keyboard.o vt.o run.o tools.o log.o menu.o menuentry.o edit.o scan_syslinux.o scan_lilo.o scan_grub.o scan_grub2.o devices.o
 
 
-# Add -m32 on 64bit OS
 UNAME := $(shell uname -m)
 
-ifeq ($(UNAME), x86_64)
-M = -m32
-endif
-
 DATE := $(shell date)
 
 
 # Set compile flags
-CFLAGS 	= -g -c -Wunused -Wuninitialized -O2 -march=i486 $(M)
-LDFLAGS = -Wall -D_FILE_OFFSET_BITS=64 --static -pthread $(M)
+CFLAGS 	= -g -c -Wunused -Wuninitialized -O2
+LDFLAGS = -Wall -D_FILE_OFFSET_BITS=64 --static -pthread
 LINK 	= $(CC) -o
 
 
-- 
2.19.1

